sched/sched/sched_timerexpiration.c

```diff
@@ -100,12 +100,30 @@ static clock_t g_timer_tick;
  * that just expired.  The value zero means that no timer was active.
  */
 
-static unsigned int g_timer_interval;
+static atomic_t g_timer_interval;
 
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
 
+static inline_function clock_t  get_time_tick(void)
+{
+#ifdef CONFIG_SYSTEM_TIME64
+  return atomic64_read((FAR atomic64_t *)&g_timer_tick);
+#else
+  return atomic_read((FAR atomic_t *)&g_timer_tick);
+#endif
+}
+
+static inline_function clock_t xchg_time_tick(clock_t tick)
```

Comment:
```suggestion
static inline_function clock_t update_time_tick(clock_t tick)
```
or exchange_time_tick

Reply:
done